src/App.js

```diff
@@ -1,5 +1,51 @@
+import { MissionUtils } from '@woowacourse/mission-utils'
+import { randomNum } from './utils/randomNum.js'
+import { userInput } from './utils/userInput.js'
+import { restartGame } from './utils/restartGame.js'
+import { resultStrikeBall } from './utils/resultStrikeBall.js'
+import { message } from './const/message.js'
 class App {
-  async play() {}
+	async play() {
+		//시작
+		MissionUtils.Console.print('숫자 야구 게임을 시작합니다.')
```

Comment:
```suggestion
MissionUtils.Console.print(MESSAGE.GAMESTART)
```
요기 메세지도 상수화해도 좋을 것 같아요!

Reply:
오~🥲🥲🥲 이런거 봐주셔서 감사합니다!